src/components/keyboard/keyboard.js

```diff
@@ -0,0 +1,83 @@
+import useSound from 'use-sound';
+import { useEffect } from 'react';
+
+import keys from "../../sounds/keys.mp3";
+
+const useKeyboardBindings = (play, setKey, splash, keys) => {
+    useEffect(() => {
+      if(!splash) {
+        const keyHandler = e => {
+          if(document.activeElement.id !== 'bpmInput' && keys.includes(e.key)) {
+            play({ id: e.key });
+            setKey(e.key);
+    
+            const element = document.getElementById('key-' + e.key);
+    
+            if(element.classList.contains('keyPressed')) {
+                element.style.animation = 'none';
+                setTimeout(function(){ 
+                    element.style.animation = '';
+                }, 1); 
+            } else {
+              element.classList.add('keyPressed');
+            }
+          }
+        }
+  
+        window.addEventListener('keydown', keyHandler);
+    
+        return () => {
+          window.removeEventListener('keydown', keyHandler);
+        };
+      }
+    }, [play, setKey, splash, keys]);
+  };
+
+function Keyboard(props) {
+    const ordered = ['$', 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 'a', 'b', 'B', 'c', 'C', 'd', 'D', 'e', 'E',
+                     'f', 'g', 'G', 'h', 'H', 'i', 'I', 'j', 'J', 'k', 'l', 'L', 'm', '!', '%', '(',
+                     '*', '@', '^', 'n', 'o', 'O', 'p', 'P', 'q', 'Q', 'r', 's', 'S', 't', 'T', 'u',
+                     'v', 'V', 'w', 'W', 'x', 'y', 'Y', 'z', 'Z'];
+
+    const keyboard = ['1', '!', '2', '@', '3', '4', '$', '5', '%', '6', '^', '7', '8', 
```

Comment:
A bit redundant, could be refactored into one list. However, we would need to change the key samples.